utils/__tests__/userData.test.ts

```diff
@@ -0,0 +1,36 @@
+import { aServicePreference } from "../../__mocks__/mocks";
+import { AllUserData } from "../userData";
+import * as E from "fp-ts/lib/Either";
+
+const mockedUserData: AllUserData = {
+  messageContents: [],
+  messageStatuses: [],
+  messagesView: [],
+  messages: [],
+  notifications: [],
+  notificationStatuses: [],
+  profiles: [],
+  servicesPreferences: [aServicePreference]
+};
+
+const mockedUserDataWithAdditionalProperty = {
+  ...mockedUserData,
+  servicesPreferences: [{ ...aServicePreference, foo: 1, bar: "hello" }]
+};
+
+describe("servicePreference decoding", () => {
+  it("should remove additional properties", () => {
+    const result = AllUserData.decode(mockedUserDataWithAdditionalProperty);
+
+    expect(E.isRight(result)).toBeTruthy();
+    if (E.isRight(result)) {
+      expect(result.right).toMatchObject(mockedUserData);
+      expect(
+        (result.right.servicesPreferences[0] as any).foo
```

Comment:
Maybe you could just check that they are equal to `aServicePreference`.

Reply:
done in https://github.com/pagopa/io-functions-admin/pull/191/commits/14030001890cd960f5309d3f6bac471151235df0